ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://json-schema.org/draft-07/schema",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Why must be http?

Reply:
It must be http because that is what the file content itself is advertising as its self-url (`$id` field). Until the official URL is moved to https, we need to use that form.

Reply:
There is also a thread on https://json-schema.slack.com/archives/C5CF75URH/p1677766289988399?thread_ts=1677763627.735939&cid=C5CF75URH which you might find useful.